modules/docker-mirror/main.tf

```
@@ -100,14 +128,23 @@ resource "google_compute_firewall" "ssh" {
   }
 }
 
+resource "random_id" "compute_address_static" {
+  prefix      = var.resource_prefix
+  byte_length = 6
+}
+
 resource "google_compute_address" "static" {
   address_type = "INTERNAL"
   subnetwork   = var.subnet_id
-  name         = "sg-executor-docker-registry-mirror"
+  name         = random_id.compute_address_static.hex
```

```suggestion
  name         = "${random_id.compute_address_static.hex}-docker-registry-mirror"
```

Maybe? Makes it a bit easier in the UI to see what it's for